BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -180,6 +180,37 @@ extension StackElement {
             child: child
         )
     }
+
+    /// Adds a `Spacer` element to the stack, which will grow (and shrink) depending on the amount
+    /// of left over available space when measuring the elements.
+    public mutating func addFlexibleSpacer() {
```

Comment:
Should we allow passing in a `minimumSize: CGFloat = 1`? Also, should the default be `0`?

Reply:
It needs to be `1` because the value is multiplied by the row in order to scale it – if we pass 0, the math becomes `something *0`, which becomes 0. Passing a minimum size seems good, too

Reply:
IMO, 1 technically works but it's usually not the right call. That is, if you have flexible space you also typically want it to have a decent looking minimum (of, say, 8pts or whatever's appropriate for your design).

Reply:
> in order to scale it

Oh, right, I forgot `Spacer` would be scaled just like any other Element in a stack... I guess I intuitively expected `Spacer` to behave more like a one-off `minimumSpacing` that can optionally flex to fill space, but not scale proportionally with other `Element`s in a stack. Hm...